backend/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The sizing logic here is a bit odd (this is half of a 1080 screen):
![image](https://github.com/user-attachments/assets/51c7c133-313d-4d8c-af7f-3c9492f1e040)

Reply:
Behaviour also seems to change  for no clear reason depending on whether I upload a pdf or dwg, the drawing side panel only pops up automatically if I upload a pdf:

![24-08-27 10-28-13](https://github.com/user-attachments/assets/e292afe7-6553-43b6-a214-962cff27add6)